vagrant/environments/pe/config.yaml

```diff
@@ -35,8 +35,8 @@ puppet:
   environment: production     # Puppet environment to link to local control-repo
 
 # Puppet Enterprise settings
-  pe_version: '2017.2.0'         # Version of PE to install on the puppet server. See Note 2
-  pe_download_root: 'https://s3.amazonaws.com/pe-builds/released/2017.2.0' # Download base url. See Note 2
+  pe_version: '2018.1.0'         # Version of PE to install on the puppet server. See Note 2
```

Comment:
Pls, Add this also in the lab environment.

Reply:
done